### PR TITLE
add userScalable option

### PR DESCRIPTION
**Pull request by williamboman**

### CODE REVIEW

File: index.ejs [hunk 1]
```diff
@@ -32,7 +32,11 @@
     <% } %>
 
     <% if (htmlWebpackPlugin.options.mobile) { %>
-    <meta content="width=device-width, initial-scale=1" name="viewport">
+    <% var viewportMetaContents = ["width=device-width", "initial-scale=1"] %>
```
Comment (jaketrent) [inline index.ejs:35]: Can this not be done via the `meta` option today?

Comment (williamboman) [reply index.ejs:35]: You mean setting `mobile: false` and adding a:

```js
{
    name: 'viewport',
    content: 'width=device-width, initial-scale=1, user-scalable=no'
}
```
?

Comment (jaketrent) [reply index.ejs:35]: Yes, essentially.

Comment (williamboman) [reply index.ejs:35]: I didn't think of that. That's much better than dealing with combinatorics.

Comment (jaketrent) [reply index.ejs:35]: Awesome, glad it works for you.

**Comment by jaketrent** [comment]
Thanks for helping work through this use case.

**Pull request closed without merging**